src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,31 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private double volume;
+    private double maxPower;
+    private double energyCapacity;
+    private EngineType engineType;
+    private double distanceToService;
+    private boolean check;
+
+    public Engine() {
+        engineType = EngineType.GASOLIN;
+        volume = 2.2;
+        maxPower = 10;
+        energyCapacity = 1000;
+        distanceToService = 10000;
+        check = false;
```

Comment:
each time when you will create a new Engine it would be always the same engine(with the same parameters). How can we change it?

Reply:
This is default constructor to initialize all fields with something. 
There are few possible options to deal with:
1. I can initialize with random values in range, f.e. volume = 1.7 + Math.rand(); or choosing random of pre-seted...
2. I can use builder to create new objects with different parameters,
3. I can use constructor with arguments to build smth, new each time, but firstly i supposed to implement it.

Reply:
Also i could add some setters and/or getters, but but don`t sure it`s necessary here..